### PR TITLE
Editorial changes to implementation report

### PR DESCRIPTION
**Pull request by agbeltran**
This PR includes my updated affiliation and some other editorial changes to the implementation report.

### CODE REVIEW

**Comment by agbeltran** [comment]
View here: https://raw.githack.com/w3c/dxwg/agb-editorial/dcat-implementation-report/index.html

Differences to current report here: https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fw3c.github.io%2Fdxwg%2Fdcat-implementation-report%2F&doc2=https%3A%2F%2Fraw.githack.com%2Fw3c%2Fdxwg%2Fagb-editorial%2Fdcat-implementation-report%2Findex.html

**Review by pwin — APPROVED** [review]
agreed

File: dcat-implementation-report/index.html [hunk 1]
```diff
@@ -28,19 +28,19 @@ <h3>Methodology</h3>
...
       </section>
       <section id="exit-criteria">
         <h3>Meeting the exit criteria</h3>
-        <p>As described in the <abbr title="Dataset Exchange Working Group">DXWG</abbr> <a href="https://www.w3.org/2017/dxwg/charter">charter</a>, to move on to Proposed Recommendation, evidence will be adduced in order to demonstrate that each of the changes made to the original DCAT Recommendation [[VOCAB-DCAT]] has been published and consumed independently in catalogs and related systems at least once, although a higher number is expected for the majority of terms.</p>
+        <p>As described in the <a href="https://www.w3.org/2017/dxwg/charter"></a><abbr title="Dataset Exchange Working Group">DXWG</abbr> charter</a>, to advance to Proposed Recommendation, evidence will be adduced in order to demonstrate that each of the changes made to the original DCAT Recommendation [[VOCAB-DCAT]] has been published and consumed independently in catalogs and related systems at least once, although a higher number is expected for the majority of terms.</p>
```
Comment (andrea-perego) [inline dcat-implementation-report/index.html]: I would keep the link to the charter, as allows readers to directly check the success criteria for DCAT, which are different from the usual ones of having 2 independent implementations.

**Review by andrea-perego — CHANGES_REQUESTED** [review]
I would keep the link to the charter, as allows readers to directly check the success criteria for DCAT, which are different from the usual ones of having 2 independent implementations.

File: dcat-implementation-report/index.html (hunk 1 above)
Comment (agbeltran) [reply dcat-implementation-report/index.html]: my intention was to keep the link, only to expand it to the phrase 'DXWG charter', will revise

Comment (agbeltran) [reply dcat-implementation-report/index.html]: @andrea-perego now back to the original link to 'charter'

**Review by andrea-perego — APPROVED** [review]
Thanks, @agbeltran .

**Pull request merged**